### PR TITLE
Usermanagement

### PR DESCRIPTION
**Pull request by RahulAadroja**
User Listing Grid Search and Export Functionality

### CODE REVIEW

File: motadata.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.k-grid-toolbar {
```
Comment (gunjankothari) [inline motadata.css]: - Don't use style at the root level.
- We will keep all style at the component level. 
- No CSS only LESS.

File: src/admin/pages/users/usersInfoGrid.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                                   :title="'Actions'"
+                                   :width="150"></kendo-grid-column>
+            </kendo-grid>
+        </div>
+    </div>
+
+</template>
+
+<script>
+    import JsZip from 'jszip';
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue:56]: Why do you need JsZip?

File: src/admin/pages/users/usersInfoGrid.vue [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    window.JSZip = JsZip;
+    export default {
+        data () {
+            return {
+                tempStore : this.$store.state.usersStore.state.users,
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue]: Pass this data from parent. Think this component as reusable component.

File: src/admin/pages/users/usersInfoGrid.vue [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+              console.log("Checkbox Toggle !!!")
+            },
+            userGridFilter(searchText) {
+                const searchCriteria = ['UserName', 'Role', 'AssignedGroups', 'Email', 'Description'];
+                const searchTextLower = searchText.toLowerCase();
+                return this.tempStore.filter( user => {
+                    return searchCriteria.some( field => {
+                        return user[field].toLowerCase().indexOf(searchTextLower) > -1
+                    });
+                });
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue:99]: I have simplified this logic for search.

File: src/layouts/sidebarRight.vue [hunk 5]
```diff
@@ -5,14 +5,14 @@
                 Header Should come here.
             </slot>
         </div>
-        <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
+      <!--  <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
```
Comment (gunjankothari) [inline src/layouts/sidebarRight.vue]: Uncomment this

File: src/main.js [hunk 6]
```diff
@@ -6,13 +6,26 @@ import 'es6-promise/auto';
...
+import '@progress/kendo-theme-default/dist/all.css'
+import '../motadata.css'
 import store from './store';
-
 import App from './App'
 import router from './router'
 import './plugins'
 
 
+import { Grid, GridInstaller } from '@progress/kendo-grid-vue-wrapper'
```
Comment (gunjankothari) [inline src/main.js]: Move Kendo import and use in ./plugins. We will keep this seprate from main

File: motadata.css (hunk 1 above)
Comment (RahulAadroja) [reply motadata.css]: Ok

File: src/admin/pages/users/usersInfoGrid.vue (hunk 2 above)
Comment (RahulAadroja) [reply src/admin/pages/users/usersInfoGrid.vue:56]: JsZip used for kendo grid export functionality.
If i don't used window.JSZip = JsZip;  then export functionality don't work.

File: src/admin/pages/users/usersInfoGrid.vue (hunk 3 above)
Comment (RahulAadroja) [reply src/admin/pages/users/usersInfoGrid.vue]: ok i wiil try to pass this data in parent component

File: src/admin/pages/users/usersInfoGrid.vue (hunk 4 above)
Comment (RahulAadroja) [reply src/admin/pages/users/usersInfoGrid.vue:99]: Nice , I like it.

File: src/layouts/sidebarRight.vue (hunk 5 above)
Comment (RahulAadroja) [reply src/layouts/sidebarRight.vue]: Ok

File: src/admin/pages/users/usersInfoGrid.vue (hunk 2 above)
Comment (gunjankothari) [reply src/admin/pages/users/usersInfoGrid.vue:56]: okay. We will later try to bind it to local scope.

**Comment by gunjankothari** [comment]
There are a few minor things we need to correct. But overall, it looks good to merge.

**Pull request merged**